integration

```diff
@@ -144,6 +144,7 @@
   "bacco007/sensor.opennem",
   "bacco007/sensor.waternsw",
   "bairnhard/fishing_assistant",
+  "bairnhard/ha-tplink-monitor",
```

Comment:
https://github.com/bairnhard/ha-tplink-monitor/blob/master/hacs.json#L3
That filename is not correct, remove it and create a new release before marking this as ready again.